src/datumaro/cli/commands/explore.py

```diff
@@ -42,12 +46,24 @@ def build_parser(parser_ctor=argparse.ArgumentParser):
     )
 
     parser.add_argument("target", nargs="?", help="Target dataset")
-    parser.add_argument(
-        "-q",
-        "--query",
-        dest="query",
-        required=True,
-        help="Image path or id of query to explore similar data",
+    query_parser = parser.add_argument_group("Query options")
+    query_parser.add_argument(
+        "--query-item-id",
+        default=None,
+        type=str,
+        help="",
+    )
+    query_parser.add_argument(
+        "--query-img-path",
+        default=None,
+        type=str,
+        help="",
+    )
+    query_parser.add_argument(
+        "--query-str",
+        default=None,
+        type=str,
+        help="",
```

Comment:
Please introduce https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group.
It seems that we need only one of them.